dist/index.html

```diff
@@ -8,8 +8,27 @@
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="parent-container">
+      <!-- <h1 class="center">Hello World!</h1> -->
+      <div class="navigation-container">
```

Comment:
**Lines 13-21:**

In our designer's mockups, it appears that the text, "CUSTOM DESIGNS - MARKETING - ..." is grouped together with the navbar. 

Perhaps these two DOM elements should be contained within the same `<div>`? This would allow use to position them on the screen together when it comes time to worry about the page's layout.

Reply:
Alright. I have combined them into the same <div> and now they are inline. I will work on figuring out how to have the text below the navbar.

Reply:
I was able to nest the text "CUSTOM DESIGNS - MARKETING - ..." within the navbar div. I believe changing parent container from inline, to block enabled me to position the  text below the navbar.